.rubocop.yml

```diff
@@ -36,7 +36,7 @@ RSpec/MultipleExpectations:
     - 'spec/requests/homeworks_spec.rb'
 
 RSpec/ExampleLength:
-  Max: 7
```

Comment:
You can disable this cop if you like; we talked about it yesterday when working on the rails template and decided this sort of thing makes more sense to discuss during code review than to have a style enforcement for.